source/jormungandr/jormungandr/parking_space_availability/bss/tests/jcdecaux_test.py

```diff
@@ -64,41 +65,64 @@ def parking_space_availability_jcdecaux_support_poi_test():
     invalid_poi = {'properties': {}}
     assert not provider.support_poi(invalid_poi)
 
+
 def parking_space_availability_jcdecaux_get_informations_test():
     """
-    Jcdecaux validate return good stands informations or None if an error occured
+    The service returns real stand informations or stand with status='Unavailable' if an error occured
     """
     webservice_response = {'2': {
         'available_bike_stands': 4,
-        'available_bikes': 8
+        'available_bikes': 8,
+        'status': 'OPEN'
     }}
     provider = JcdecauxProvider(u"vélib'", 'Paris', 'api_key', {'jcdecaux'})
     provider._call_webservice = MagicMock(return_value=webservice_response)
-    assert provider.get_informations(poi) == Stands(4, 8)
+    assert provider.get_informations(poi) == Stands(4, 8, 'Open')
     provider._call_webservice = MagicMock(return_value=None)
-    assert provider.get_informations(poi) is None
+    assert provider.get_informations(poi) == Stands(0, 0, 'Unavailable')
     invalid_poi = {}
-    assert provider.get_informations(invalid_poi) is None
+    assert provider.get_informations(invalid_poi) == Stands(0, 0, 'Unavailable')
+
 
 def parking_space_availability_jcdecaux_get_informations_unauthorized_test():
     """
-    Jcdecaux validate return None if not authorized
+    The service returns real stand informations or stand with status='Unavailable' if not authorized
     """
     webservice_unauthorized_response = {
         'error': 'Unauthorized'
     }
     provider = JcdecauxProvider(u"vélib'", 'Paris', 'unauthorized_api_key', {'jcdecaux'})
     provider._call_webservice = MagicMock(return_value=webservice_unauthorized_response)
-    assert provider.get_informations(poi) is None
+    assert provider.get_informations(poi) == Stands(0, 0, 'Unavailable')
+
+
+def parking_space_availability_jcdecaux_get_informations_with_status_Closed_test():
+    """
+    The service returns real stand informations or stand with status='Unavailable' if an error occured
```

Comment:
same: return real`time` stand information(no `s`)

You may also just remove it, as it's enough in the part where we deal with the feed